docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-snippet/icon-snippet.component.html

```diff
@@ -0,0 +1,15 @@
+<uxd-snippet class="icon-code-snippet" [code]="code" [synchronous]="true"></uxd-snippet>
+
+<button uxFocusIndicator
+        [uxTooltip]="tooltipTemplate"
+        [showTriggers]="['mouseenter']"
+        (click)="copy()"
+        type="button"
+        class="btn btn-link btn-icon button-secondary snippet-copy-button"
+        aria-label="Copy to clipboard">
+    <ux-icon name="clipboard" aria-hidden="true"></ux-icon>
```

Comment:
Remove aria-hidden, I don't think it's needed for the icon since it has no text. If it turns out that it is needed then we can add aria-hidden directly to the ux-icon component itself.